catalog/ui/search-ui/search-endpoint/src/main/java/org/codice/ddf/ui/searchui/query/service/UserService.java

```diff
@@ -76,8 +76,8 @@ public void getUser(final ServerSession remote, Message message) {
                 String username = SubjectUtils.getName(subject);
                 userMap.put("username", username);
                 if (subject instanceof ddf.security.Subject) {
-                    userMap.put("isAnonymous",
-                            String.valueOf(((ddf.security.Subject) subject).isAnonymous()));
+                    userMap.put("isGuest",
```

Comment:
It looks like you changed the JSON here, but you didn't change any of the corresponding JS that reads this. This is going to break the Search UI.